src/scales/scale.time.js

```diff
@@ -338,7 +338,7 @@ module.exports = function(Chart) {
 			var me = this;
 			if (!value || !value.isValid) {
 				// not already a moment object
-				value = moment(me.getRightValue(value));
+				value = me.parseTime(me.getRightValue(value));
```

Comment:
If I did my maths correctly, it currently saves 4KB (~3%) on the minified version.